### ISSUE TITLE
Pivot: Narrator rapidly reads all of the headers when selecting a header

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#1396]
1. Open pivot.html from http://1drv.ms/1LlOiH5 in Edge
2. Open Narrator and put focus on the `Toggle Pivot Width` button
3. Hit capslock+right-arrow a couple of times to focus the `important` header
4. Hit capslock+spacebar to select the `important` header

**Unexpected result**: Narrator rapidly reads out all of the headers: "flagged important deleted block inbox unread group". It's difficult to make meaning of this Narrator announcement.

**Expected result**: Narrator should announce something more reasonable like the name of the selected header and the fact that it is now selected. See how Narrator interacts with the XAML Pivot.
